riemann_client/client.py

```diff
@@ -152,6 +153,9 @@ def create_dict(event):
                 value = list(value)
             elif descriptor.name == 'attributes':
                 value = dict(((a.key, a.value) for a in value))
+            elif descriptor.type == \
+                    google.protobuf.descriptor.FieldDescriptor.TYPE_FLOAT:
+                value = round(value, 1)
```

Comment:
It seems like rounding here made the tests pass, as the test value "11.1" has some float encoding precision loss.